docs/actions-reusable.md

```diff
@@ -0,0 +1,63 @@
+Like [scripted actions](actions-scripts.md), reusable actions are logical units of analytic code.
+However, whereas the source code for a scripted action must be copied from study to study, the source code for a reusable action is shared by different studies.
+
+## Running reusable actions
+
+You can browse existing reusable actions at <https://actions.opensafely.org>.
+Although each is different, they have a common API.
+Consider the following extract from a study's *project.yaml*:
+
+```yaml
+actions:
+  generate_my_cohort:
+    # ...
+
+  run_a_reusable_action:
+    # The reusable action is called "a_reusable_action".
+    # We will run version "v1.0.0".
+    run: a_reusable_action:v1.0.0
+    needs: [generate_my_cohort]
+    outputs:
+      moderately_sensitive:
+        output: output/output_from_a_reusable_action.csv
+```
+
```

Comment:
Is it worth a bit more prose and/or examples and/or a link to safetab to explain how actions can also have configuration?

Reply:
Eventually, yes 🙂 For now, I wanted to move the documentation from the issue to a more visible, permanent location. `safetab` also needs attention. Whilst it's working as a reusable action (opensafely-actions/test-reusable-actions#1), I'd prefer to improve its documentation and fix some inconsistencies first.